XenModel/Messages.resx

```diff
@@ -10479,7 +10494,7 @@ Please reconnect the host and try again</value>
     <value>This pool has no shared storage</value>
   </data>
   <data name="POOL_HAS_PV_GUEST_WARNING" xml:space="preserve">
-    <value>{0}: Support for paravirtualized (PV) guests is dropped as of [XenServer] {1}. Click "Learn more" to see the list of supported guest operating systems. </value>
+    <value>{0}: Support for paravirtualized (PV) guests has been removed in [XenServer] {1}. Click Learn more to see the list of supported guest operating systems. </value>

```

Comment:
While testing this I've noticed XenCenter doesn't show this warning because of a format exception introduced in 
https://github.com/xenserver/xenadmin/commit/f7391e991b7f53c67084c48cc2d18d2117a4a4e3#diff-759b56e08e904e5eed5cac3cfc9b5fdbL53

Reply:
See https://github.com/xenserver/xenadmin/pull/2615